src/main/java/com/codecool/dungeoncrawl/ui/enemiesmove/GhostMove.java

```diff
@@ -0,0 +1,28 @@
+package com.codecool.dungeoncrawl.ui.enemiesmove;
+
+import com.codecool.dungeoncrawl.data.CellType;
+import com.codecool.dungeoncrawl.data.GameMap;
+
+public class GhostMove implements  EnemiesMoves{
```

Comment:
Structure names in Java are singulare. `EnemiesMoves` should be `EnemyMove` or `EnemyMover`